### PR TITLE
Merge

### PR DESCRIPTION
**Pull request by ferazo94**

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -2,27 +2,121 @@ const mongoose = require('mongoose');
...
\ No newline at end of file
+const User = mongoose.model('User', userSchema, 'users');
+
+const eventSchema = Schema({
+  name: String,
+  location: String,
+  date: Date,
+  description: String,
+  organizer: String,
+  group_id: Number,
```
Comment (aywgit) [inline database/index.js:45]: Should this be array of group NAMEs, since they are unique?

File: database/index.js [hunk 2]
```diff
@@ -2,27 +2,121 @@ const mongoose = require('mongoose');
...
+});
```
Comment (aywgit) [inline database/index.js:46]: Need array of users attending (just First name and Last name in a string)

**Review by erichand18 — APPROVED** [review]

File: database/index.js (hunk 1 above)
Comment (ferazo94) [reply database/index.js:45]: yes lets doi it in the next merge

File: database/index.js (hunk 2 above)
Comment (ferazo94) [reply database/index.js:46]: yes lets do it in the next merge

**Review by aywgit — APPROVED** [review]

**Pull request merged**